conanfile.py

```diff
@@ -99,6 +99,12 @@ def _setup_args_gn(self):
         self._set_env_arg(args, "CXXFLAGS", "extra_cflags_objcc")
         self._set_env_arg(args, "LDFLAGS",  "extra_ldflags")
 
+        cxx = os.getenv("CXX")
+        if cxx and "g++" in cxx:
```

Comment:
```suggestion
        if self.settings.compiler == "gcc":
```
Does it work like this ?